app/views/patients/_abortion_information.html.erb

```diff
@@ -35,6 +37,16 @@
         <% end %>
       </div>
     </div>
+
+    <div class="row">
+      <div class="col-sm-12" id="clinic_finder_container">
+
+        <%= link_to display_clinic_finder_path, class: 'display_button', remote: true do %>
+        <h3>Find clinics</h3>
```

Comment:
I'm thinking of this from a testing perspective. If we can just do assert_texts, then this shouldn't be an issue. I will stick with the current solution until I find some other alternatives.